src/parkapi_sources/converters/herrenberg_bike/models.py

```diff
@@ -145,36 +146,32 @@ class HerrenbergBikePropertiesInput(ValidataclassMixin):
     max_stay: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
     fee_description: OptionalUnset[str] = NoneToUnsetValue(StringValidator(max_length=512)), DefaultUnset
 
+    def to_dict(self, *args, **kwargs) -> dict:
+        ignore_keys = ['type', 'original_uid', 'supervision_type', 'date_surveyed']
+        result = {key: value for key, value in super().to_dict(*args, **kwargs).items() if key not in ignore_keys}
```

Comment:
Minor issue: I am thinking about if renaming `result` as `new_dictionary` or something like that would make the code a bit better understandable.

Reply:
I would keep it that way as a shorter variable name helps to stay in one line which keeps overview, and it is the result of this methos, so the name is not wrong